bankdroid-legacy/src/main/java/com/liato/bankdroid/banking/banks/AbsIkanoPartner.java

```diff
@@ -125,8 +125,8 @@ public Urllib login() throws LoginException, BankException, IOException {
     @Override
     public void update() throws BankException, LoginException, BankChoiceException, IOException {
         super.update();
-        if (username == null || password == null || username.length() == 0
-                || password.length() == 0) {
+        if (getUsername() == null || getPassword() == null || getUsername().length() == 0
```

Comment:
Would it make sense to have getUsername/getPassword (or some other more suitable function) "normalize" empty username/password so that they're either always null or always with a length of 0?

So that one of "string == null" or "string.length() == 0" could be removed.